package.json

```diff
@@ -1,8 +1,10 @@
 {
-  "name": "netlify-plugins",
+  "name": "@netlify/plugins-list",
   "private": true,
   "version": "1.0.0",
-  "description": "Netlify plugins",
+  "description": "List of Netlify plugins",
+  "main": "site/plugins.json",
+  "files": [],
```

Comment:
`main` is always included in `files` even when `files` is an empty array.
I double-checked with `npm publish --dry`.